### DISCUSSION TITLE
Sign actions after login

### DISCUSSION THREAD

**Discussion by dominictobias** [go-webauthn/webauthn#290] [Q&A]
Hi I would like to sign privileged actions after the user has logged in, e.g. the user wants to delete some data so I create a deletion action and the user signs it with their webauthn, then another user approves the action by signing it with theirs so it can execute.

Is this something the lib can handle? Do I use the same Login methods as the examples for it? Is that what `ClientDataJSON` is for? Thanks

**Comment by james-d-elliott** [comment]
It's not something I've specifically investigated. Is there a spec for this?
